checker/jdk/Makefile.jdk

```diff
@@ -30,8 +30,8 @@ build: classes
 
 ############## Validate annotated jdk against missing methods ##############
 
-SOURCES = $(shell find src/java src/javax -name '*.java')
-# CLASSES = $(shell find src/java src/javax -name '*.class')
+SOURCES = $(shell find src/java src/javax src/org/w3c src/org/xml -name '*.java')
```

Comment:
Shouldn't this be "src/org src/com" to include the two directories that are currently not included?